scripts/inference/README.md

```diff
@@ -175,6 +188,7 @@ python convert_hf_to_onnx.py --pretrained_model_name_or_path local/path/to/huggi
 Please open a Github issue if you discover any problems!
 
 ## Converting an HF MPT to FasterTransformer
```

Comment:
Do we want to keep FT conversion scripts? afaik, https://github.com/NVIDIA/FasterTransformer is not maintained anymore by Nvidia and wont be going forward.